web/widgets.py

```
@@ -135,3 +139,220 @@ def __init__(self, attrs=None):
+                        if (newItem) {{
+                            newItem.querySelector('input').focus();
+                        }}
+                    }});
+                    // Initial render
+                    renderItems();
+                }});
+            </script>
+        </div>
+        """
+
+    def value_from_datadict(self, data: dict, files: dict, name: str) -> list:
```

Ruff (0.8.2)

Unused method argument: `files`

(ARG002)